.github/workflows/deploy-sage-account-web.yml

```diff
@@ -40,7 +43,7 @@ jobs:
         uses: aws-actions/configure-aws-credentials@v2
         with:
           aws-region: us-east-1
-          role-to-assume: "arn:aws:iam::797640923903:role/sagebase-github-oidc-sage-ProviderRolesynapsewebmo-1DNSZNE6O57RM"
+          role-to-assume: arn:aws:iam::797640923903:role/sagebase-github-oidc-sage-ProviderRolesynapsewebmo-1DNSZNE6O57RM
```

Comment:
please update due to PR https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/859